extensions/wikia/UserProfilePageV3/js/UserProfilePage.js

```diff
@@ -389,7 +394,10 @@ var UserProfilePage = {
 					window.location = UserProfilePage.reloadUrl;
 				}
 			},
-			error: UserProfilePage.error
+			error: function (msg) {
```

Comment:
...or you can leave 'success' and 'error' callbacks as there were (well, maybe with your upgrades for coding conventions :)) and use 'complete' callback to setup this saveButton again.